### PR TITLE
New tools: jeb-android, jeb-arm, jeb-intel, jeb-mips, jeb-webasm

### PR DESCRIPTION
**Pull request by osospeed**

### CODE REVIEW

File: packages/jeb-android/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# See COPYING for license details.
+
+pkgname=jeb-android
+pkgver=3.0.beta.X
+pkgrel=1
+pkgdesc='Android decompiler.'
+arch=('any')
+groups=('blackarch' 'blackarch-reversing' 'blackarch-mobile')
+url='https://www.pnfsoftware.com/jeb/android'
+license=('custom:unknown')
```
[the remark this answers is not in the document]
Comment (osospeed) [reply packages/jeb-android/PKGBUILD]: Unfortunately did find a clear one. I've sent an Email inquiring about this.
Will fix the rest once I get a response ...

**Comment by FFY00** [comment]
Why were the license fields removed?? This won't be merged until the license issue is fixed.

**Comment by osospeed** [comment]
Me:
> In the license field I put 'custom:unknown' as license type. Do you want specify one or should I leave it as it is ?"


Nicolas Falliere (JEB Decompiler, Founder and Architect, nico@pnfsoftware.com):
>The "key name" field is pure metadata, you may leave it blank if you want to.

**Comment by FFY00** [comment]
I am sorry but we need actual permission to use it, either by a public email or by a compatible license. Also, I don't follow the response, what is the `key name` field and what does it has to do with the license?

**Comment by FFY00** [comment]
Were the emails signed by any chance? I still think it shouldn't be merged without a public statement. We could make it avaliable via blackman only if that's possible. Either eay, is @noptrix wants to merge this, he can go ahead.

**Comment by Edu4rdSHL** [comment]
I send a email directly to PNF software about that, i'm waiting for response.

**Comment by Edu4rdSHL** [comment]
I can confirm what @osospeed says, i received a email from PNF Software that allow we to include the demos in the tools.

@osospeed please fix the license fields according to https://www.pnfsoftware.com/jeb/LICENSE.TXT

**Comment by noptrix** [comment]
sup guys. can we merge this in? :)

**Comment by FFY00** [comment]
If you feel comfortable with it then yes.

**Comment by osospeed** [comment]
So, should we put `license=('custom:unknown')` in the PKGBUILD ?

**Comment by noptrix** [comment]
> So, should we put `license=('custom:unknown')` in the PKGBUILD ?

yes. but i can take that.

**Pull request merged**